### PR TITLE
Adding openvino-telemetry

### PR DESCRIPTION
**Pull request by ilya-lavrenov**
@conda-forge/help-python

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please ping https://github.com/orgs/conda-forge/teams/help-python.

This recipe PR is ready for review.

**Comment by ilya-lavrenov** [comment]
https://github.com/orgs/conda-forge/teams/help-python, ready for review!

**Comment by ilya-lavrenov** [comment]
@conda-forge/help-python, ready for review!

File: recipes/openvino-telemetry/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "openvino-telemetry" %}
+{% set version = "2022.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (ocefpaf) [inline recipes/openvino-telemetry/meta.yaml:9]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/openvino-telemetry/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "openvino-telemetry" %}
+{% set version = "2022.3.0" %}
```
Comment (ocefpaf) [inline recipes/openvino-telemetry/meta.yaml:2]: Latest version is 2023.1.0 but that one seems to be missing a sdist on PyPI. Can you open an issue upstream to request one?

File: recipes/openvino-telemetry/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  entry_points:
+    - opt_in_out = openvino_telemetry.opt_in_out:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - requests >=2.20.0
```
Comment (ocefpaf) [inline recipes/openvino-telemetry/meta.yaml:23]: You should not need the run time dependency at build time but it seems that this package imports itself at build time :-/

PS: can you open an issue upstream about this? It is bad practice to require a local installation to be able to build a package.

File: recipes/openvino-telemetry/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    - openvino_telemetry.utils
+  commands:
+    - opt_in_out --help
+
+about:
+  home: "https://github.com/openvinotoolkit/telemetry"
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: "OpenVINO™ Telemetry package for sending statistics with user's consent, used in combination with other OpenVINO™ packages."
```
Comment (ocefpaf) [inline recipes/openvino-telemetry/meta.yaml:41]:
```suggestion
  home: https://github.com/openvinotoolkit/telemetry
  license: Apache-2.0
  license_family: Apache
  license_file: LICENSE
  summary: OpenVINO™ Telemetry package for sending statistics with user's consent, used in combination with other OpenVINO™ packages.
```

**Pull request merged**